### ISSUE TITLE
Training with only one class, but the test data does not guarantee any target is always available.

### ISSUE THREAD

**Issue by SongWang2017** [wizyoung/YOLOv3_TensorFlow#175]
Hi there:
Thanks for implementing this terrific repo! 
At the moment, I'm training a single class object detection on my own data.
With only the single class name put into the classes.name file.
After around 100 epoch training, the precision is still unreasonably low which is around 0.01, that means the system always tries to predict something, even though sometimes there are no object available.
So should I just put some empty background images into the training set? if yes, the training require at least one target in the image, is it ok to put some random regions in the background image as bounding box and declare them as a new "background" class?

Or is there any idea how to modify the source code especially the loss function, so that  the system can accomdate images with no label (I mean there is no object in the whole image)? 

best,
